### ISSUE TITLE
Pull Request

### ISSUE THREAD

**Issue by jimmyheaddon** [davidfowl/NuGetPowerTools#20]
Hi David,

I have to admit that I'm a complete noob when it comes to GitHub, but I have a few additions that I'd like to push your way around items like:

Add-PropertyGroup
Add-ItemGroup
Remove-PropertyGroup
Remove-ItemGroup

Let me know if you're interested.

Cheers!

**Comment by davidfowl** [comment]
Sounds good. Did you implement them using msbuild api

**Comment by jimmyheaddon** [comment]
Yeah I've followed in your footsteps as far as I can see, but some of the nicer methods from Microsoft.Build.BuildEngine (like Project.RemovePropertyGroupsWithMatchingCondition()) have been deprecated in Microsoft.Engine.Evaluation so I've provided alternate methods that work but are worth reviewing.
